tox.ini

```diff
@@ -91,10 +91,6 @@ envlist =
     mysql_contrib-py{27,35,36,37,38,39}-mysqlconnector{80,}
     mysqldb_contrib-py27-mysqldb{12,}
     mysqldb_contrib-py{27,35,36,37,38,39}-mysqlclient{13,14,}
-    psycopg_contrib-py{27,35,36}-psycopg2{24,25,26,27,28}
-    psycopg_contrib-py37-psycopg2{27,28}
-# psycopg <2.7 doesn't support Python 3.8: https://github.com/psycopg/psycopg2/issues/854
-    psycopg_contrib-py{38,39}-psycopg2{28}
     pylibmc_contrib-py{27,35,36,37,38,39}-pylibmc{140,150,}
     pylons_contrib-py27-pylons{096,097,010,10,}
     pymemcache_contrib{,_autopatch}-{py27,py35,py36,py37,py38}-pymemcache{130,140}
```

Comment:
You should also remove the dependencies list like `pyscopg224` etc below.

Reply:
Unfortunately that requires moving sqlalchemy over as well. Will do that in a separate PR.